libraries/AP_Scheduler/AP_Scheduler.cpp

```diff
@@ -198,3 +206,66 @@ float AP_Scheduler::load_average() const
     const uint32_t used_time = loop_us - (_spare_micros/_spare_ticks);
     return used_time / (float)loop_us;
 }
+
+void AP_Scheduler::loop()
+{
+    // wait for an INS sample
+    AP::ins().wait_for_sample();
+
+    const uint32_t timer = AP_HAL::micros();
+
+    // used by PI Loops
+    G_Dt                    = (float)(timer - loop_start) / 1000000.0f;
```

Comment:
I am a little mixed about the large division but * 0.000001f doesn't look much better